### PR TITLE
Added publications, dataset, and tool

### PR DESCRIPTION
**Pull request by rmcgranaghan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,6 +183,8 @@ Before working with SDO data make sure to checkout the [Guide to SDO Data Analys
 - [SpaceML](http://spaceml.org/repo) - A Machine Learning toolbox and developer community building the next generation AI applications for space science and exploration containing a set of [code examples](https://github.com/spaceml-org/helionb-sdoml).
 - [Sunpy](https://sunpy.org/) - Set of tools for solar data analysis with Python.
 - [SWPC CME Analysis Tool (SWPC_CAT)](https://ccmc.gsfc.nasa.gov/swpc_cat_web/) - Primary tool being used by NOAA SWPC in measuring key parameters of a Coronal Mass Ejection (CME) [code](https://github.com/nasa/ccmc-swpc-cat-web)
+- [The Heliophysics KNOWledge Network](https://github.com/rmcgranaghan/Helio-KNOW)
```
Comment (mariusgiger) [inline README.md:186]: this is already listed on line 110 :)

File: README.md [hunk 2]
```diff
@@ -161,6 +162,8 @@ Before working with SDO data make sure to checkout the [Guide to SDO Data Analys
 - Ivanov, Sergey, et al. "Solar activity classification based on Mg II spectra: towards classification on compressed data." arXiv preprint arXiv:2009.07156 (2020). [link](https://arxiv.org/pdf/2009.07156.pdf)
 - Wang, Jingjing, et al. "Solar Flare Predictive Features Derived from Polarity Inversion Line Masks in Active Regions Using an Unsupervised Machine Learning Algorithm." The Astrophysical Journal 892.2 (2020): 140. [link](https://iopscience.iop.org/article/10.3847/1538-4357/ab7b6c/pdf)
 - Ahmadzadeh, Azim, et al. "How to Train Your Flare Prediction Model: Revisiting Robust Sampling of Rare Events." The Astrophysical Journal Supplement Series 254.2 (2021): 23. [link](https://iopscience.iop.org/article/10.3847/1538-4365/abec88/pdf)
+-  McGranaghan, R. M., Mannucci, A. J., Wilson, B. D., Mattmann, C. A., & Chadwick, R. (2018). New capabilities for prediction of high-latitude ionospheric scintillation: A novel approach with machine learning. Space Weather, 16, 1817– 1846. https://doi.org/10.1029/2018SW002018
```
Comment (mariusgiger) [inline README.md:165]: currently the papers are sorted in ascending order by publication date and use MLA citation style. Could you please adjust it accordingly?

McGranaghan, Ryan M., et al. "New capabilities for prediction of high‐latitude ionospheric scintillation: A novel approach with machine learning." Space Weather 16.11 (2018): 1817-1846. [link](https://doi.org/10.1029/2018SW002018)

McGranaghan, Ryan M., et al. "Toward a next generation particle precipitation model: Mesoscale prediction through machine learning (a case study and framework for progress)." Space Weather 19.6 (2021): e2020SW002684. [link](https://doi.org/10.1029/2020SW002684)

File: README.md [hunk 3]
```diff
@@ -132,6 +132,7 @@ Before working with SDO data make sure to checkout the [Guide to SDO Data Analys
 - Angryk, Rafal A., et al. "Multivariate time series dataset for space weather data analytics." Scientific data 7.1 (2020): 1-13. [link](https://www.nature.com/articles/s41597-020-0548-x.pdf) [code](https://bitbucket.org/gsudmlab/workspace/projects/FP)
 - Shneider, Carl, et al. "A Machine-Learning-Ready Dataset Prepared from the Solar and Heliospheric Observatory Mission." arXiv preprint arXiv:2108.06394 (2021). [link](https://arxiv.org/abs/2108.06394)
 - Baek, Ji-Hye, et al. "Solar Event Detection Using Deep-Learning-Based Object Detection Methods." Solar Physics 296.11 (2021): 1-15. [link](https://link.springer.com/article/10.1007/s11207-021-01902-5)
+- McGranaghan, Ryan; ryan.mcgranaghan@colorado.edu; https://orcid.org/0000-0002-9605-0007; Mannucci, Anthony; http://orcid.org/0000-0003-2391-8490; Mattmann, Chris; et al. (2018): Machine learning databases used for Journal of Geophysical Research: Space Physics manuscript: "New capabilities for prediction of high-latitude ionospheric scintillation: A novel approach with machine learning.". figshare. Dataset. https://doi.org/10.6084/m9.figshare.6813131.v1
```
Comment (mariusgiger) [inline README.md:135]: please convert links to markdown links and use the MLA style for the citation

**Review by mariusgiger — CHANGES_REQUESTED** [review]
Hi Ryan, thanks a lot for your contribution! Indeed very interesting work that you are pursuing and I like the initiative with Helio-KNOW. Unfortunately, I have only found it yesterday when searching the Helionauts forum, otherwise this would have been a great source for me.

I made a few minor comments. After that this PR can be merged.

Cheers and have a nice weekend,
Marius

File: README.md (hunk 3 above)
Comment (rmcgranaghan) [reply README.md:135]: McGranaghan, R., et al. "Machine learning databases used for Journal of Geophysical Research: Space Physics manuscript: New capabilities for prediction of high-latitude ionospheric scintillation: A novel approach with machine learning.". (2018). figshare. Dataset. [link](https://doi.org/10.6084/m9.figshare.6813131.v1)

File: README.md (hunk 2 above)
Comment (rmcgranaghan) [reply README.md:165]: Thanks for the correct formatting - sorry for the oversight on my end. Feel free to update the references!

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**